concent_api/core/views.py

```diff
@@ -32,6 +33,25 @@ def send(request, message):
         save_messages_to_database(data['type'], data['message_task_to_compute'], request.body)
         return HttpResponse("ACCEPTED", status = 202)
 
+    elif data['type'] == "MessageAckReportComputedTask":
+        validate_message_task_to_compute(data['message_task_to_compute'])
+
+        current_time = int(datetime.datetime.now().timestamp())
+
+        if current_time <= data['message_task_to_compute']['deadline'] + settings.CONCENT_MESSAGING_TIME:
+
+            task_to_compute_exists  = Message.objects.filter(task_id = data['message_task_to_compute']['task_id'], type = "MessageForceReportComputedTask")
+            other_ack_message       = Message.objects.filter(task_id = data['message_task_to_compute']['task_id'], type = "MessageAckReportComputedTask")
+            reject_message          = Message.objects.filter(task_id = data['message_task_to_compute']['task_id'], type = "MessageRejectReportComputedTask")
+            if not task_to_compute_exists:
+                print("No MessageForceReportComputedTask in database.")
+                return HttpResponse("No MessageForceReportComputedTask in database.", status = 204)
+            if other_ack_message or reject_message:
+                print("Task already Ack or Reject.")
+                return HttpResponse("Task already Ack or Reject.", status = 204)
```

Comment:
`HTTP 204 NO CONTENT` means that the operation succeeded and there is no content returned in the body of the response. So why are you putting a message in the body?